### DISCUSSION TITLE
Timestamp feature for saved media

### DISCUSSION THREAD

**Discussion by thracianx** [rodit/SnapMod#108] [Ideas & Feature Suggestions]
Having name of the snap sender or story along with a timestamp(MM:DD:YYYY-Hrs/Min/Sec) would be helpful in the next update if possible because I save a lot of premium content so having a way to organize by name would be super helpful.

**Comment by RevealedSoulEven** [comment]
What about along with the name of the user at tthe beginning too? If will help the file manager app to list them according to their owners

**Reply by rodit** [reply]
The name of the user is already included by default if you enable 'Enable Custom Snap Download Path'

**Comment by rodit** [comment]
I agree - I will add a parameter (like %d) for the configurable save paths for stories etc which will be a formatted date. However, you will have to change the path to contain %d yourself after the update eventually comes if you have previously modified the save path manually from SnapMod's settings.

**Comment by rodit** [comment]
See v1.6.1
